### PR TITLE
Simple styling

### PR DESCRIPTION
**Pull request by myiahm**
Revamped form also added ACH payment option

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const path = require('path')
 const open = require('open')
 const port = process.env.port || 5000
 
-const tilledSecretApiKey = 'sk_SECRET_KEY_HERE'
+const tilledSecretApiKey = 'sk_14blwMvv8p1zfA7vF1MhANksnECiPsYM2FIWqS6iNiDf0rhD0I4GFARSkejTlodLcBhBTJ5l9qol2GYs435yYHvvDviXG0VMe0lX'
```
Comment (mcook08) [inline app.js:8]: don't commit the secret key. You'll need to do a new branch without the secret key in here.

File: index.html [hunk 2]
```diff
@@ -54,88 +67,108 @@
...
 
   <!-- Included here for simplicity of example -->
   <script>
+
     document.addEventListener('DOMContentLoaded', async () => {
-      const tilledAccountId = 'acct_ACCOUNT_ID_HERE'
+      const tilledAccountId = 'acct_u2LZeK0EZu85WLVxObS5T'
       const tilled = new Tilled(
-        'pk_PUBLISHABLE_KEY_HERE', 
+        'pk_EqUYyPl8PhrIjYGtZU3Xn3lD6QgoZ9ehp0fp2GsurimvXFyQOyXtNny2AhGTxufLDa3MLLaoqw9TQGA2Yk2atcDzGbnMXJuJ0SPF',
```
Comment (mcook08) [inline index.html:171]: same with publishable key and account id

File: index.html [hunk 3]
```diff
@@ -244,6 +275,90 @@ <h3 class="panel-title display-td">Payment Details</h3>
 
       })
   }) // end of DOMContentLoaded
+  document.addEventListener('DOMContentLoaded', async () => {
+      const tilledAccountId = 'acct_u2LZeK0EZu85WLVxObS5T'
+      const tilled = new Tilled(
+        'pk_EqUYyPl8PhrIjYGtZU3Xn3lD6QgoZ9ehp0fp2GsurimvXFyQOyXtNny2AhGTxufLDa3MLLaoqw9TQGA2Yk2atcDzGbnMXJuJ0SPF', 
+        tilledAccountId,
```
Comment (mcook08) [inline index.html:282]: account_id and pk

**Review by mcook08 — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (myiahm) [reply index.html:282]: Fixed creating new PR now!

**Pull request closed without merging**